PF/PFAssignment2/HexCalc/HexCalc.java

```diff
@@ -0,0 +1,194 @@
+package GET2018.PF.PFAssignment2.HexCalc;
+
+/**
+ * This class implements various methods on Hexadecimal No
+ * 
+ * @author Mohit Sharma
+ */
+public class HexCalc {
+	String hexaDecimal;
+
+	public HexCalc(String hexaDecimal) {
+		if (hexaDecimal.equals(null)) {
```

Comment:
This condition itself will throw null pointer exception. because you can not call any method on null value. It should be like this:
if (hexaDecimal == null) {
			throw new NullPointerException("HexaDecimal No is Null");
		}